cocos/base/CCConsole.cpp

```diff
@@ -108,10 +108,12 @@ namespace {
         if (Director::getInstance()->getOpenGLView())
         {
             HWND hwnd = Director::getInstance()->getOpenGLView()->getWin32Window();
-            SendMessage(hwnd,
-                        WM_COPYDATA,
-                        (WPARAM)(HWND)hwnd,
-                        (LPARAM)(LPVOID)&myCDS);
+            // use non-block version of SendMessage 
```

Comment:
Could you double check docs:
https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-sendnotifymessagea
Are you sure about SendNotifyMessageA?
"If the window was created by the calling thread, SendNotifyMessage calls the window procedure for the window and does not return until the window procedure has processed the message"
Maybe PostMessage should be used?

Reply:
`PostMessage` seems more "non-block" than `SendNotifyMessage`, thanks for your correcting .